locales/zh_CN/LC_MESSAGES/scalar_tensor.po

```diff
@@ -28,62 +29,60 @@ msgstr "定义"
 
 #: ../../scalar_tensor.rst
 msgid "Parameters"
-msgstr ""
+msgstr "参数"
 
 #: ../../scalar_tensor.rst:12
 msgid "(DataType) type of the tensor element"
 msgstr "（数据类型）张量元素的数据类型"
 
 #: ../../scalar_tensor.rst:13
 msgid "(optional, scalar or tuple) the shape of tensor"
-msgstr "（可选，标量或tuple）张量的形状"
+msgstr "（可选，标量或元组）张量的形状"
 
 #: ../../scalar_tensor.rst:14
-#, fuzzy
 msgid "(optional, scalar or tuple) see :ref:`offset`"
-msgstr "（可选，标量或tuple）张量的形状"
+msgstr "（可选，标量或元组）请参见 :ref:`offset`"
 
 #: ../../scalar_tensor.rst:16
 msgid ""
-"For example, this creates a *dense* tensor with four ``int32`` as "
-"elements: ::"
-msgstr "例如，下列代码将创建由4个 ``int32`` 组成的密集张量： ::"
+"For example, this creates a *dense* tensor with four ``int32`` as elements: ::"
+msgstr "例如，这将创建一个具有四个 ``int32`` 作为元素的 *稠密(dense)* 张量： ::"
 
 #: ../../scalar_tensor.rst:19
 msgid "x = ti.var(ti.i32, shape=4)"
-msgstr ""
+msgstr "x = ti.var(ti.i32, shape=4)"
 
 #: ../../scalar_tensor.rst:21
 msgid "This creates a 4x3 *dense* tensor with ``float32`` elements: ::"
-msgstr "下列代码将创建由 ``float32`` 组成的4x3密集张量： ::"
+msgstr "这将创建一个元素为 ``float32`` 类型的4x3 *稠密* 张量： ::"
 
 #: ../../scalar_tensor.rst:24
 msgid "x = ti.var(ti.f32, shape=(4, 3))"
-msgstr ""
+msgstr "x = ti.var(ti.f32, shape=(4, 3))"
 
 #: ../../scalar_tensor.rst:26
 msgid ""
-"If shape is ``()`` (empty tuple), then a 0-D tensor (scalar) is created: "
-"::"
-msgstr "如果形状设为 ``()`` （空元组），那将创建一个0维张量（即标量）： ::"
+"If shape is ``()`` (empty tuple), then a 0-D tensor (scalar) is created: ::"
+msgstr "如果 shape 是 ``()`` （空元组），则创建一个0-D张量（标量）： ::"
 
 #: ../../scalar_tensor.rst:29
 msgid "x = ti.var(ti.f32, shape=())"
-msgstr ""
+msgstr "x = ti.var(ti.f32, shape=())"
 
 #: ../../scalar_tensor.rst:31
 msgid "Then access it by passing ``None`` as index: ::"
-msgstr "然后以 ``None`` 为指标来访问它： ::"
+msgstr "随后通过传递 ``None`` 作为索引来访问它： ::"
 
 #: ../../scalar_tensor.rst:34
 msgid "x[None] = 2"
-msgstr ""
+msgstr "x[None] = 2"
 
 #: ../../scalar_tensor.rst:36
 msgid ""
-"If shape is **not provided** or ``None``, the user must manually "
-"``place`` it afterwards: ::"
-msgstr "如果 **不提供** 形状参数 ``shape`` ，那么稍后用户必须手动放置（place）那个张量： ::"
+"If shape is **not provided** or ``None``, the user must manually ``place`` it "
+"afterwards: ::"
+msgstr ""
+"如果形状是 **not provided（未提供）** 或 **None（空）**，则用户必须在其后手动 ``place（放置）`` 形状： ::"
```

Comment:
```suggestion
"如果形状参数 **未提供** 或指定为 ``None``，则其后用户必须在手动放置 (place) 它： ::"
```
We don't translate python keywords like `None`.

Reply:
got it thx :rofl: didn't get ur point the first time